### PR TITLE
feature/user-account-configuration

### PR DESCRIPTION
**Pull request by Marpop**

### CODE REVIEW

File: logs/templates/logs/base.html [hunk 1]
```diff
@@ -1,6 +1,13 @@
 <p>
 	<a href="{% url 'logs:index' %}">Daily log</a> -
-	<a href="{% url 'logs:entries' %}">Entries</a>
+	<a href="{% url 'logs:entries' %}">Entries</a> -
```
Comment (k4ju) [inline logs/templates/logs/base.html]: try not to hard-code any strings, use translations instead
https://docs.djangoproject.com/en/1.10/topics/i18n/translation/

File: users/templates/users/login.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{% extends "logs/base.html" %}
+
+{% block content %}
+
+	{% if form.errors %}
```
Comment (k4ju) [inline users/templates/users/login.html]: what's wrong with indentation here?

Comment (Marpop) [reply users/templates/users/login.html]: I don't know.  Everything between '{% block content %}' and '{% endblock content %}' are indented.

**Comment by Marpop** [comment]
Nope, I don't bother :)

File: logs/templates/logs/base.html (hunk 1 above)
Comment (Marpop) [reply logs/templates/logs/base.html]: Thanks 👍

File: users/templates/users/login.html (hunk 2 above)
Comment (k4ju) [reply users/templates/users/login.html]: but isn't two times more than it should be? ;)

Comment (Marpop) [reply users/templates/users/login.html]: no, I will check it

Comment (Marpop) [reply users/templates/users/login.html]: on my computer, everything is fine

**Pull request merged**